lib/simplifyExpression/basicsSearch/index.js

```diff
@@ -85,6 +88,20 @@ function removeExponentByOne(node) {
   return Node.Status.noChange(node);
 }
 
+// If `node` is of the form 1^x, reduces it to a node of the form 1.
+// Returns a Node.Status object.
+function removeExponentBaseOne(node) {
+  if (node.op === '^' &&                         // an exponent with
+      checks.resolvesToConstant(node.args[1]) && // a power not a symbol and
+      Node.Type.isConstant(node.args[0]) &&      // a constant base
+      node.args[0].value === '1') {              // of value 1
+    let newNode = clone(node.args[0]);
```

Comment:
I need to eventually add this to the linter, and we don't do this right in a bunch of places in the codebase already heh, but this can be `const` instead of `let` cause you don't change it